lighthouse-core/gather/gatherers/anchor-elements.js

```diff
@@ -59,6 +61,7 @@ function collectAnchorElements() {
 
     return {
       href: resolveURLOrEmpty(node.href.baseVal),
+      rawHref: node.getAttribute('href') || '',
```

Comment:
I guess there's no name equivalent in this situation? I don't entirely remember what this was supposed to catch if it's not an `HTMLAnchorElement` 🤔

Reply:
> I guess there's no name equivalent in this situation? I don't entirely remember what this was supposed to catch if it's not an `HTMLAnchorElement` 🤔 

It was for SVG (`SVGAElement`), originally #6483 then through https://github.com/GoogleChrome/lighthouse/pull/7101#discussion_r254879958. We probably should have left a comment :)

It doesn't look like `SVGAElement` ever had a `name` attribute, though of course you could just add one. Not sure if it's worth tracking since it's not accidentally usable or once used and now obsolete :)

Reply:
Slightly above I see:

```ts
  /** @type {Array<HTMLAnchorElement|SVGAElement>} */
  const anchorElements = getElementsInDocument('a');
```

https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.svgaelement.html

Maybe not worth adding a `name`

Reply:
ah yeah I didn't scroll enough that sounds familiar now, agreed no need for name it can stay optional 👍